source/ed/connectors/fusio_parser.h

```diff
@@ -161,6 +162,15 @@ struct FrameFusioHandler : public GenericHandler {
         "frame_start_date", "frame_end_date"}; }
 };
 
+struct DatasetFusioHandler : public GenericHandler {
+    DatasetFusioHandler(GtfsData& gdata, CsvReader& reader) : GenericHandler(gdata, reader) {}
+    int id_c, contributor_c, start_date_c, end_date_c, type_c, desc_c, system_c;
+    void init(Data&);
+    void handle_line(Data& data, const csv_row& row, bool is_first_line);
+    const std::vector<std::string> required_headers() const { return {"dataset_id", "contributor_id",
```

Comment:
I think it's easier to only use the DatasetFusioHandler and remove the FrameFusioHandler no ?